lib/ChangeDueDateDialog/ChangeDueDate.js

```diff
@@ -11,7 +11,7 @@ import DueDatePicker from './DueDatePicker';
 import LoanList from './LoanList';
 import css from './ChangeDueDateDialog.css';
 
-const DueDateUnchangeableItemStatuses = ['Declared lost', 'Claimed returned'];
+const DueDateUnchangeableItemStatuses = ['Declared lost', 'Claimed returned', 'Aged to lost'];
```

Comment:
Since we add items to the array for each new status, wouldn't it be better to move it into a constants file, as we already do in other components?

Reply:
Sure ... I didn't bother here because the scope is limited to a single class and there are no other similar constants, but it could be good for future expansion.